#! /bin/sh -e
## 07_bad_font_drawing.dpatch by Eduard Bloch <blade@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: comments to the rudicolous rants against Debian packages

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
       -patch) patch -f --no-backup-if-mismatch -p1 < $0;;
       -unpatch) patch -f --no-backup-if-mismatch -R -p1 < $0;;
	*)
		echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
		exit 1;;
esac

exit 0
				    
@DPATCH@
diff -urNad ./doc/rxvt.7.pod /tmp/dpep-work.AoTIrT/rxvt-unicode-4.8/doc/rxvt.7.pod
--- ./doc/rxvt.7.pod	2005-01-18 02:26:11.000000000 +0100
+++ /tmp/dpep-work.AoTIrT/rxvt-unicode-4.8/doc/rxvt.7.pod	2005-01-23 01:17:38.000000000 +0100
@@ -89,14 +89,10 @@
 
 =item Why are the secondary screen-related options not working properly?
 
-Make sure you are using C<TERM=rxvt-unicode>. Some pre-packaged
-distributions (most notably Debian GNU/Linux) break rxvt-unicode
-by setting C<TERM> to C<rxvt>, which doesn't have these extra
-features. Unfortunately, some of these (most notably, again, Debian
-GNU/Linux) furthermore fail to even install the C<rxvt-unicode> terminfo
-file, so you will need to install it on your own (See the question B<When
-I log-in to another system it tells me about missing terminfo data?> on
-how to do this).
+Make sure you are using C<TERM=rxvt-unicode> and the appropriate terminfo file
+is installed. On Debian GNU/Linux, see README.Debian.gz for further details.
+Also see the question B<When I log-in to another system it tells me about
+missing terminfo data?> on how to do this).
 
 =item Rxvt-unicode does not seem to understand the selected encoding?
 
